cmd/controller-manager/app/options/options.go

```diff
@@ -94,7 +94,7 @@ type Options struct {
 	// It can be set to "0" to disable the metrics serving.
 	// Defaults to ":8080".
 	MetricsBindAddress string
-	// concurrentClusterSyncs is the number of cluster objects that are
+	// ConcurrentClusterSyncs is the number of cluster objects that are
```

Comment:
nice catch!

But for others, I'm sorry I didn't make it clear.
For comments not for exported variables/functions/methods, there is no rule for should comment start with uppercase or not, so I guess no needs to update them.

Reply:
I agree with u,I didn't update them